distros_test.go

```diff
@@ -57,56 +68,123 @@ func Focal(ctx context.Context, t *testing.T, client *dagger.Client) *dagger.Con
 		"systemd", "strace", "ssh", "udev", "iptables",
 	).
 		WithExec([]string{"systemctl", "enable", "ssh"}).
-		WithExec([]string{"systemctl", "enable", "systemd-udevd"}).
 		WithExec([]string{"update-alternatives", "--set", "iptables", "/usr/sbin/iptables-legacy"}).
 		WithMountedFile("/tmp/packages-microsoft-prod.deb", deb).
 		WithExec([]string{"/usr/bin/dpkg", "-i", "/tmp/packages-microsoft-prod.deb"}).
 		WithNewFile("/opt/moby/install.sh", dagger.ContainerWithNewFileOpts{Contents: debInstall, Permissions: 0744})
 }
 
 func Bionic(ctx context.Context, t *testing.T, client *dagger.Client) *dagger.Container {
-	c := client.Container().From(targets.BionicRef)
 	deb := client.HTTP("https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb")
-	c2 := c.WithMountedFile("/tmp/packages-microsoft-prod.deb", deb).
-		WithExec([]string{"/usr/bin/dpkg", "-i", "/tmp/packages-microsoft-prod.deb"})
-	c2 = apt.Install(c2, client.CacheVolume(targets.BionicAptCacheKey), client.CacheVolume(targets.BionicAptLibCacheKey), "aptly")
-	return c.WithRootfs(c2.Rootfs())
+
+	c := client.Container().From(targets.BionicRef)
+	return apt.Install(c, client.CacheVolume(targets.BionicAptCacheKey), client.CacheVolume(targets.BionicAptLibCacheKey),
+		"systemd", "strace", "ssh", "udev", "iptables",
+	).
+		WithExec([]string{"systemctl", "enable", "ssh"}).
+		WithExec([]string{"update-alternatives", "--set", "iptables", "/usr/sbin/iptables-legacy"}).
+		WithMountedFile("/tmp/packages-microsoft-prod.deb", deb).
+		WithExec([]string{"/usr/bin/dpkg", "-i", "/tmp/packages-microsoft-prod.deb"}).
+		WithNewFile("/opt/moby/install.sh", dagger.ContainerWithNewFileOpts{Contents: debInstall, Permissions: 0744})
 }
 
 func Bullseye(ctx context.Context, t *testing.T, client *dagger.Client) *dagger.Container {
-	c := client.Container().From(targets.BullseyeRef)
 	deb := client.HTTP("https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb")
-	c2 := c.WithMountedFile("/tmp/packages-microsoft-prod.deb", deb).
-		WithExec([]string{"/usr/bin/dpkg", "-i", "/tmp/packages-microsoft-prod.deb"})
-	c2 = apt.Install(c2, client.CacheVolume(targets.BullseyeAptCacheKey), client.CacheVolume(targets.BullseyeAptLibCacheKey), "aptly")
-	return c.WithRootfs(c2.Rootfs())
+
+	c := client.Container().From(targets.FocalRef)
```

Comment:
This one too (should be `targets.BullseyeRef`)

Reply:
🤦 
I've looked over these so many times.
I did a double check before pushing just now and found even another one.

Anyway, should be good now.